src/viam/services/motion/client.py

```diff
@@ -232,6 +284,24 @@ async def stop_plan(
     ):
         """Stop a component being moved by an in progress ``move_on_globe()`` or ``move_on_map()`` call.
 
+        ::
+
+            motion = MotionClient.from_robot(robot=robot, name="builtin")
+            my_base_resource_name = Base.get_resource_name("my_base")
+
+            # Assuming a move_on_globe started started the execution
```

Comment:
Is it possible to link to the method here, rather than commenting out what could've started this process? It's kind of confusing to have another method's usage in the docs of this method

Reply:
Like [down here](https://github.com/viamrobotics/viam-python-sdk/pull/550/files#diff-3f50c1b6e65f9668bfb199b019f5969113233f18cee420aa0ade2142218bc2e2R437-R438), you don't write a full `connect` function

Reply:
Not sure if adding an explanation that connect is in code sample is necessary, since *technically* people can use Viam without ever going to the web app

Reply:
Sorry, but I meant the opposite! I like how you don't include the `connect` function in the example because that function is not part of the example. The assumption below is that the user already has the robot, we're just showing them how to use that specific function.

I would like the same thing to happen here. You have a lot of extra code for getting a movement sensor, setting a destination, and calling `move_on_globe`, when those aren't part of this example. In my opinion, it would be better if you simply linked to that function in the example. Something like:

```python
# Assuming [MotionClient.move_on_globe](LINK TO FUNCTION) started the execution
```

and then remove the rest of the commented-out code of obtaining a movement sensor, setting the destination, and calling the move_on_globe function.